### PR TITLE
add chat page

### PR DESCRIPTION
**Pull request by smartdev0809**

### CODE REVIEW

**Comment by charmhorse** [comment]
okay

**Comment by dot425** [comment]
add main side bar

File: src/pages/chat/bottomSide.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        ></input>
+        <span className=" text-xl text-minionBlue cursor-pointer">
+          <FaPaperclip></FaPaperclip>
+        </span>
+      </div>
+      <button className="px-5 py-3 bg-minionBlue text-white text-[20px] border-2 border-white rounded-xl cursor-pointer hover:text-minionBlue hover:border-minionBlue hover:bg-white duration-300 ease-linear">
+        <FaPaperPlane></FaPaperPlane>{" "}
+      </button>
+    </div>
+
```
Comment (pandablue0809) [inline src/pages/chat/bottomSide.jsx]: change css
not match design

**Review by pandablue0809 — APPROVED** [review]
LGTM

**Pull request merged**